### PR TITLE
chore: remove CI on node v13.x

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

Title says it all. While I was here I renamed the job to be more descriptive.

### Context of Change

We don't use node 13.x anymore. It's also deprecated and we shouldn't be using it anymore.

### Type of Change

- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

### TypeScript/Hooks Update

N/A

## Test Plan

CI passes.

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  build-and-test:
```
Comment (JST5000) [inline .github/workflows/nodejs.yml:13]: Why does this command need to change?

Comment (mvadari) [reply .github/workflows/nodejs.yml:13]: That's just the name of the job. I renamed it to be more descriptive. It's not just building, it's building and testing.

**Review by JST5000 — APPROVED** [review]

**Review by intelliot — APPROVED** [review]
Would it make sense to add Node.js 16.x or 17.x now?

**Comment by mvadari** [comment]
> Would it make sense to add Node.js 16.x or 17.x now?

Yes, but that was failing (you can see the results in the commit history of this PR) and it wasn't worth the time to figure out what was going wrong right now.

**Pull request merged**